src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.DataLake.Analytics" version="3.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DataLake.Analytics" version="3.1.2-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@ro-joowan similar comment about remove the new references in this file. These references can be found in the `packages.config` file in the `Commands.Common.ResourceManager` project.

Reply:
Done - thanks!